gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/HadoopFileSystemTestBase.java

```diff
@@ -444,16 +442,11 @@ public void testConcurrentCreationWithOverwrite_bothSucceed() throws Exception {
                         hadoopPath, text2, /* numWrites= */ 1, /* overwrite= */ true)));
     executorService.shutdown();
 
-    // Wait until result will be ready
-    assertThat(futures.get(0).get()).isEqualTo(text1.length());
-    assertThat(futures.get(1).get()).isEqualTo(text2.length());
-
```

Comment:
We need to replace this with assert of succeeded and failed/ignored write, it could be similar to [this](https://github.com/GoogleCloudDataproc/hadoop-connectors/blob/b9416d5f03e18d99117c36de0fc91d12b5ce12cb/gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/HadoopFileSystemTestBase.java#L407:L416).